utxo-info/README.md

```diff
@@ -0,0 +1,19 @@
+```
+shortname: ?/UTXO-IMPL
+name: A short discription of the current UTXO implementation
+type: informational
+status: Draft
+editor: Vanshdeep Singh <vanshdeep@bigchaindb.com>
+```
+
+## Description
+The UTXO (unspent transaction output) of each transaction is tracked using the a merkle tree. The merkle root of this tree is used as the app hash and retuned to Tendermint during `commit`. Below is a short summary of the UTXO is calculated,
```

Comment:
It's not clear how all the outputs (which are like floating independent dots) are somehow connected to form a Merkle tree. Maybe draw a picture? They say pictures are worth 1000 words and in this case I think it's true.